content/blog/2020-07-07-cml-release.md

```diff
@@ -0,0 +1,182 @@
+---
+title: 'New Release: Continuous Machine Learning (CML) is CI/CD for ML'
+date: 2020-07-07
+description: |
+  The DVC team is releasing a new open-source project called Continuous Machine
+  Learning. CML helps to organize MLOps infrastructure on top of the traditional
+  software engineering stack.
+
+descriptionLong: |
+  Today, the DVC team is releasing a new open-source project called Continuous
+  Machine Learning, or CML (https://cml.dev) to mainstream the best engineering
+  practices of CI/CD to AI and ML teams. CML helps to organize MLOps
+  infrastructure on top of the traditional software engineering stack.
+
+picture: 2020-07-07/cover.png
+pictureComment: CML release
+author: dmitry_petrov
+commentsUrl: https://discuss.dvc.org/t/dvc-1-0-release/412
+tags:
+  - Release
+  - CI/CD for ML
+  - MLOps
+  - DataOps
+---
+
+## 1. CI/CD for machine learning problems
+
+Continuous integration and continuous delivery (CI/CD) is a widely-used software
+engineering practice. It's a validated approach to increasing the agility of
+software development without sacrificing stability. **But why haven't CI/CD
+practices taken root in machine learning and data science so far?**
+
+We see three substantial technical barriers to using standard CI systems with
+machine learning projects:
+
+1. **Data dependencies.** In ML, data plays a similar role as code: ML results
+   critically depend on datasets, and changes in data need to trigger feedback
+   just like changes in source code. Furthermore, multi-GB datasets are
+   challenging to manage with Git-centric CI systems.
+2. **Metrics-driven.** The traditional software engineering idea of pass/fail
+   tests does not apply in ML. As an example, `+0.72% accuracy` and
+   `-0.35% precision` does not answer the question if the ML model is good or
+   not. Detailed reports with metrics and plots are needed to make a good/bad
+   model discussion
+3. **CPU/GPU resources**. ML training often requires more resources to train
+   then is typical to have in CI/CD runners. CI/CD must be connected with cloud
+   computing instances or Kubernetes clusters for ML training.
+
+## 2. CI/CD for ML is the next step for DVC team
+
+Since the beginning, our motivation has been helping ML teams benefit from
+DevOps. We started DVC because we knew that data management would be a crucial
+bottleneck, and sure enough, DVC was a big step towards making pipelines and
+experiments manageable and reproducible. But conversations with our community
+have brought us to one conclusion again and again: CI/CD for ML is the holy
+grail.
```

Comment:
Maybe link this
```suggestion
have brought us to one conclusion again and again:
[CI/CD](#3-continuous-machine-learning-release) for ML is the holy grail.
```
so you can skip to #3 if needed? Not sure, I just think it takes a while to get to 3 where CML is introduced (but it's mentioned in the abstract so may not be a big deal).